README.md

```diff
@@ -49,7 +59,69 @@ The `fairlearn` package contains the following algorithms for mitigating unfairn
 > _Note_:
 > DP refers to demographic parity, EO to equalized odds, and BGL to bounded group loss. For more information on these and other terms we use in this repository please refer to the [terminology page](./TERMINOLOGY.md). To request additional algorithms or fairness definitions, please open a [new issue](https://github.com/fairlearn/fairlearn/issues).
 
-## Install fairlearn
+
+### Fairlearn dashboard
+
```

Comment:
We currently give some partial examples for the dashboard, but not for the algorithms, so it feels unbalanced. Maybe we can give some simple (incomplete?) example for how to use the algorithms as well, to make things more balanced?

Reply:
I'm happy to hold this off for another PR. A more concrete suggestion:
* add a micro-example how to run one of the mitigation algorithms (say exp. grad or threshold optimizer)?
* add a micro-example how to invoke the dashboard with multiple models

Reply:
Noted for the next version of changes.